dev/breeze/doc/adr/0002-implement-standalone-python-command.md

```diff
@@ -138,7 +138,7 @@ There are a few properties of Breeze/CI scripts that should be maintained though
   run a command and get everything done with the least number of prerequisites
 
 * The prerequisites for Breeze and CI are:
-   * Python 3.8+ (Python 3.8 end of life is October 2024)
+   * Python 3.9+ (Python 3.9 end of life is October 2025)
```

Comment:
```suggestion
   * Python 3.9+ (Python 3.9 end-of-life is in October 2025)
```

Reply:
nah, I don't want to touch old ADR docs, they stay as they are once generated.